lib/gems/pending/appliance_console/external_httpd_configuration.rb

```diff
@@ -15,6 +15,7 @@ module ExternalHttpdConfiguration
       SSSD_CONFIG          = "/etc/sssd/sssd.conf".freeze
       PAM_CONFIG           = "/etc/pam.d/httpd-auth".freeze
       HTTP_KEYTAB          = "/etc/http.keytab".freeze
+      HTTP_CONFIG_FILE     = "/etc/httpd/conf.d/manageiq-redirects-ws".freeze
```

Comment:
Is the absence of this file the reason this is happening or are we looking to this file as a marker of some other condition?

I guess I'm curious if we could check something less fragile like if `httpd` is running in case we change the name of the file or something. It just seems like this file might not be the best marker as this code didn't even know about it before this change.